README.md

```diff
@@ -1,50 +1,102 @@
 
 ## Install
+
 Supported environment
-* Linux x86_64 with Python3.8 ~ 3.10
+
+- Linux x86_64 with Python3.8 ~ 3.10
 
 Command
+
 ```sh
 $ pip install qdd
 ```
 
 ## Install from source
+
 ### Build
+
 You must prepare the following software.
 
-* cmake (>=3.25)
-  * You can download the executable from [Official GitHub](https://github.com/Kitware/CMake/releases).
-  * If you use Ubuntu, [the Kitware repository](https://apt.kitware.com/) is easier to install.
-* build
-  * Just install it via pip.
+- cmake (>=3.25)
+  - You can download the executable from [Official GitHub](https://github.com/Kitware/CMake/releases).
+  - If you use Ubuntu, [the Kitware repository](https://apt.kitware.com/) is easier to install.
+- build
+  - Just install it via pip.
+
+You can manually build QDD as follows.
 
-You can build QDD as follows.
 ```sh
 $ cmake . -DCMAKE_BUILD_TYPE=Release
 $ cmake --build . -j
 $ python3 -m pip install build
 $ python3 -m build
 ```
+
 BUILD_TYPE can be either `Release`, `Debug` or `RelWithDebInfo`.
 You can find an installable wheel file in 'dist' directory.
 
-### Wheel Instllation
+Alternatively, you can just execute the shell script in the project root to build it in one line:
+
+```sh
+$ ./scripts/local_build_and_test.sh build
+```
+
+If virtualenv is not used, it will create a new one named `.venv-qdd` under the project root.
+BUILD_TYPE is set to `Release` this way.
+If you want to just build C extension module (.so file), skipping distribution building process, you can specify `cmake` as the first argument of the shell script.
+
+
+### Test (Optional)
+
+This source distribution provides functionality and performance tests with `pytest` and `./test/qdd_test`.
+
+Command
+
+```sh
+$ cmake . -DCMAKE_BUILD_TYPE=Release
+$ cmake --build . -j
+$ pip install -e .[test]
+$ pytest test
+$ ./test/qdd_test
+```
+
+If you want just to use this package, not editing itself, it is NOT recommended to execute these commands in your in-use virtualenv because it will install qdd package as editable, and qdd will not be installed under site-packages and python will try to import it from this directory whenever it is imported. In that case, it would be better to install qdd to your working virtualenv from binary distribution or without `-e` option, or execute these commands in a separated virtualenv. However, if you want to contribute to this package, it would be a nice way to keep using the environment because your edits will be reflected immediately to the package.
+
+Alternatively, you can just execute the shell script in the project root to test this package in one line:
+
+```sh
+$ ./scripts/local_build_and_test.sh test
+```
+
+If virtualenv is not used, it will create a new one named `.venv-qdd` under the project root.
+This command includes the build of C++ extension module. BUILD_TYPE is set to `Release`.
+
+If you pass all tests, please go on to create wheel file to install it as written in the section "Build" and "Wheel Installation".
```

Comment:
just a suggestion: [section links](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#section-links) are friendly for readers to navigate them to "Build" and "Wheel Installation".